docs/.island/config.ts

```diff
@@ -10,6 +10,12 @@ const getSidebar = () => ({
         { text: '实现一键复制功能', link: '/note/copy' },
       ],
     },
+    {
+      text: '无趣的',
+      items: [
+        { text: '当初为什么选择走这条路？', link: '/article/select' },
```

Comment:
I think the title of the article should be called choose

Reply:
done